src/Components/News/NewsListItem.css

```diff
@@ -0,0 +1,37 @@
+.card {
```

Comment:
If here we're styling ALL `.card` in the project, shouldn't this be in the main .css file?